packages/custom-elements/tests/html/reactions.html

```diff
@@ -1,3 +1,19 @@
+<!doctype html>
+<html>
+<head>
+<title>Custom Element Reactions</title>
+<script>
+  (window.customElements = window.customElements || {}).forcePolyfill = true;
+</script>
+<script src="../../node_modules/es6-promise/dist/es6-promise.auto.min.js"></script>
```

Comment:
Why are all these scripts new in a lot of these files?

Reply:
GitHub is showing these files as moved with additions since they're mostly the same even though they were also changed from JS to HTML. Before, they were all imported by `packages/custom-elements/tests/index.html` and run in the context of that page but now they're all separate pages.